### PR TITLE
BAEL-5785 Map.of() vs Map.ofEntries() test

### PR DESCRIPTION
**Pull request by Basler182**

### CODE REVIEW

File: core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    public void mapOf() {
+        // Use Map.of() to create an empty immutable map
+        Map<Long, String> map = Map.of();
+        assertNotNull(map);
+
+        // Use Map.of() to create an immutable map with one entry
+        Map<Long, String> map1 = Map.of(1L, "value1");
+        assertNotNull(map1);
+        assertThat(map1.size()).isEqualTo(1);
+        assertThat(map1.get(1L)).isEqualTo("value1");
```
Comment (andrzejsydor) [inline core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java:22]: you can also add test for immutability

File: core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+public class MapOfEntriesVsMapOfUnitTest {
+
+    @Test
+    public void mapOf() {
+        // Use Map.of() to create an empty immutable map
+        Map<Long, String> map = Map.of();
+        assertNotNull(map);
+
+        // Use Map.of() to create an immutable map with one entry
+        Map<Long, String> map1 = Map.of(1L, "value1");
```
Comment (andrzejsydor) [inline core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java:19]:
```suggestion
        Map<Long, String> mapWIthEntry = Map.of(1L, "value1");
```

File: core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+import java.util.Map;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class MapOfEntriesVsMapOfUnitTest {
+
+    @Test
+    public void mapOf() {
```
Comment (andrzejsydor) [inline core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java:13]: it's junit5 could works without `public`
```suggestion
    void mapOf() {
```

**Review by andrzejsydor — CHANGES_REQUESTED** [review]

File: core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java (hunk 1 above)
Comment (Basler182) [reply core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java:22]: Good point. I'll add this!

File: core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java (hunk 3 above)
Comment (Basler182) [reply core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java]: Good suggestion. Will change it.

**Pull request merged**